src/usecases/handle-figma-file-update-event-use-case.test.ts

```diff
@@ -35,8 +38,56 @@ describe('handleFigmaFileUpdateEventUseCase', () => {
 			}),
 		);
 
+		it('should log a warning and continue if saving the team name to the database fails', async () => {
+			const associatedAtlassianDesigns: Record<string, AtlassianDesign> =
```

Comment:
**nitpick:** Consider using `Map` instead of an `object` since it provides a handy API for its construction, which can simplify the code:

```ts
const associatedAtlassianDesignsById = new Map(
				associatedFigmaDesigns
					.map(figmaDesign => figmaDesign.designId.toAtlassianDesignId())
					.map(id => [id, generateAtlassianDesign({ id })])
				);
```

Reply:
Nice, I didn't know about `Map`